custom_components/tahoma/cover_entities/generic_cover.py

```diff
@@ -1,68 +1,73 @@
-"""Base class for TaHoma covers, shutters, awnings, etc."""
+"""Base class for Overkiz covers, shutters, awnings, etc."""
+from __future__ import annotations
+
+from collections.abc import Mapping
+from typing import Any, cast
+
 from homeassistant.components.cover import (
-    ATTR_POSITION,
     ATTR_TILT_POSITION,
     SUPPORT_CLOSE_TILT,
     SUPPORT_OPEN_TILT,
     SUPPORT_SET_TILT_POSITION,
     SUPPORT_STOP_TILT,
     CoverEntity,
 )
+from homeassistant.components.overkiz.entity import OverkizEntity
```

Comment:
```suggestion
from homeassistant.components.overkiz.entity import OverkizEntity
```

This is not a correct import :-).

Reply:
You saved me!

Reply:
Hehe, I also added that ugly non relative import because of codestyle rules in core..

Reply:
Good to know, thx 👍